openverse-api/catalog/api/models/media.py

```diff
@@ -54,7 +54,11 @@ class AbstractMedia(IdentifierMixin, MediaMixin, FileMixin, OpenLedgerModel):
     )
     removed_from_source = models.BooleanField(default=False)
 
-    view_count = models.IntegerField(default=0)
+    view_count = models.IntegerField(
```

Comment:
Should we remove this, too?

Reply:
I'm not sure. I made it nullable so that it gets out of the way to begin with.